apps/tasks/README.md

```diff
@@ -0,0 +1,21 @@
+# Tasks
+
+**TODO: Add description**
```

Comment:
What do I do or what am I intended to do?

Reply:
Yeah, yeah. Documentation is important.